watertap/unit_models/electrodialysis_1D.py

```diff
@@ -696,6 +696,8 @@ def eq_mass_transfer_term_concentrate(self, t, x, p, j):
             self.diluate.length_domain,
             doc="Electrical power consumption of a stack",
         )
+        # # TODO: Check this constraint formulation. Is this correct?
+        #           it states: dP/dx = b*L*u(x)*i(x)
```

Comment:
Yes, I think it is. Be noted that the power for 1D is indexed by x so, when doing costing, you'll need to use `power_electrical_x[0, self.diluate.length_domain.last()]` to represent the total power consumed through the entire length.

Reply:
Thanks for checking on this.

What I did for the power was to used the variables from 0D or 1D based on the `_get_performance_contents` function. Then I also made sure that each key in the performance table had the same name. Thus, in the Costing package, I could reference the same performance metric regardless of unit model type.

Reply:
Then it should not be of an issue since the index has been specified in the `_get_performance_contents` function.  Let me know if you ran into any related problems.